life_expectancy/main.py

```diff
@@ -2,38 +2,41 @@
 Module that contains the main function to run the process.
 """
 
+import os
 import argparse
 
 import pandas as pd
 
 from life_expectancy.cleaning import clean_data
-from life_expectancy.constants import DATA_DIR, FILE_NAME, FILE_PATH
+from life_expectancy.constants import DATA_DIR
 from life_expectancy.file_handlers import load_data, save_data
 
 
 def main(
-    file_path: str, file_name: str, data_dir_path: str, country: str = "PT"
-) -> pd.DataFrame:
+    data_dir_path: str, country: str = "PT", file_path: str = os.path.join(DATA_DIR, "eu_life_expectancy_raw.tsv")
```

Comment:
Because you are not using the data_dir_path anymore, we can exclude it from the function.

Reply:
I use this argument, inside this function when i call the function save_data. So i need this argument too